examples/replay/config-0.25-degree.yaml

```
@@ -211,3 +211,78 @@ FV3Dataset:
         - tmp
         - ugrd
         - vgrd
+
+MOM6Dataset:
+    file_prefixes:
+        - ocn_
+    
+    path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/03h-freq/zarr/"
+    coords_path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/coordinates/zarr/"
+    forecast_hours: [0]
+
+    cycles:
+        start: "1994-01-01T00"
+        end: "1999-06-13T06"
+        freq: "6h"
+
+    time:
+        start: "1993-12-31T18"
+        end: "1999-06-13T03"
```

The cycles and time will look a little different in the ocean, since it doesn't have the IAU, which modifies the forecast hour time with a -6 hour offset from the cycle. I'm happy to talk more about that if you want. It's weird.
```suggestion
        start: "1994-01-01T00"
        end: "2023-12-31T06"
```

---

done.